pkg/proxy/apis/kubeproxyconfig/v1beta1/types.go

```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package v1alpha1
+package v1beta1
```

Comment:
Is there a reason `omitempty` and `// +optional` aren't used at all here?

Reply:
Someone from @kubernetes/sig-network-api-reviews needs to go through and identify which fields are optional/omitempty